docker/subset-fastq-dataset/filterFastqByReadName.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+
+import click
+
+@click.command()
+@click.option('--in-fastq-gz',help='gz compressed input fastq file')
+@click.option('--out-fastq-gz',help='gz compressed output fastq file')
+@click.option('--keep-reads-gz',help='gz compressed file with read names to keep')
```

Comment:
Is there a reason everything is gzipped?

Reply:
I've always found that compressing fastqs almost always leads to performance increase because reading/writing such files when doing light processing is IO bound. Also helps prevent filling up the HDD.